penguin/project/models.py

```
@@ -30,6 +30,32 @@ class TaskPhase(Enum):
     BLOCKED = "blocked"      # Waiting on dependencies or clarification
 
 
+    @classmethod
+    def allowed_transitions(cls) -> Dict["TaskPhase", List["TaskPhase"]]:
```

Ruff (0.15.12)

[warning] Remove quotes from type annotation

Remove quotes

(UP037)